content/docs/command-reference/push.md

```diff
@@ -29,7 +29,7 @@ and preserving data versions (input datasets, intermediate results, models,
 these commands.
 
 The `dvc push` command allows us to upload data to remote storage. It doesn't
-save any changes in the code or DVC-files (those should be saved by using
+save any changes in the code or `.dvc` files (those should be saved by using
 `git commit` and `git push`).
```

Comment:
...any changes to the code, `dvc.yaml`, or `.dvc` files (those should be saved with...)